draft-packages/menu/docs/Menu.stories.tsx

```diff
@@ -79,6 +79,29 @@ const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
 }) => (
   <StoryWrapper isReversed={isReversed}>
     <StoryWrapper.RowHeader headings={["Default", "Icon Button"]} />
+
+    <StoryWrapper.Row rowTitle="Short list">
```

Comment:
Thanks for the update @kaydwithers - I have a few small tweaks before I give the PR the green light. Instead of  the `Short list` example being a new row, could we instead move this into the "open" and "closed" with the respective states rows so we can visually capture the these for chromatic. We'll need a title added to headings in  ln81.

With an open an close state for the shorter list our snapshots can capture what the menu looks like without the scroll bar so any change to the menu items size impact overflow can be caught in visual regression.

Shout out if you have any further queries :)

Reply:
@mcwinter07 I've added short/long list open examples, is that better?

Reply:
Main reason I'd say we should have them both in the stickersheet stories is that this is what we send to chromatic. By default our stickersheet stories are the only ones we send to chromatic to reduce the overall spend on snapshots.

Given you've done the legwork by making the individual stories I'm happy to merge this in to get your fix moving and we can use those in the updated docs when we migrate Menu KAIO (`@kaizen/components`) 👍